src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -405,8 +339,9 @@ def list_role_assignment_change_logs(cmd, start_time=None, end_time=None):
 
 
 def _backfill_assignments_for_co_admins(cli_ctx, auth_client, assignee=None):
+    worker = MultiAPIAdaptor(cli_ctx)
     co_admins = auth_client.classic_administrators.list()  # known swagger bug on api-version handling
-    co_admins = [x for x in co_admins if _get_role_property(x, 'email_address')]
+    co_admins = [x for x in co_admins if x.email_address]
```

Comment:
worker.get_role_property(x, 'email_address') ?

Reply:
and for the below lines where we refer to email_address

Reply:
no needed, classic-admin's api-version never changed.

Reply:
Thanks